Install-Dependencies.bat

```diff
@@ -262,6 +262,50 @@ call :PrintInfo "Refreshing environment variables..."
     )
 )
 
+REM ---------------------------Link python to python3--------------------------------------------------
+
+:python3Check
+setlocal enabledelayedexpansion
+@(
+    (
+        python3 --version >nul 2>&1
+    ) && (
+        call :PrintInfo "python3 is linked properly"
+        REM No need to create a symlink if python3 is already working
+        goto :installPythonStuff

```

Comment:
Need an `endlocal` command in this path also

Reply:
fixed with latest commit